<?xml version="1.0"?>

<<<<<<< HEAD
<launch>

	<!-- Publishes robot position related to map frame coordinates. Used by online_qtc_creator -->
	<node pkg="iliad_human_local_navigation" type="spencer_to_bayes.py" name="spencer_to_bayes" output="screen">
		<param name="person_topic" value="/robot4/perception/tracked_persons"/>
		<param name="target_frame" value="map_laser2d"/>
	</node>
=======
 <!--

  -->

<launch>

        <!-- Publishes robot position related to map frame coordinates. Used by online_qtc_creator -->
        <node pkg="iliad_human_local_navigation" type="spencer_to_bayes.py" name="spencer_to_bayes" output="screen">
              <param name="person_topic" value="/robot1/groundtruth/tracked_persons"/>
              <param name="target_frame" value="world"/>
        </node>




>>>>>>> c1c926f39dc66539d5ea7c65531f7e64befff6cc

</launch>
